### PR TITLE
README file updated

### PR DESCRIPTION
**Pull request by Mehulipal**
Please review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,35 @@
...
+Getting Started
+Objective
+Creating issues
+Already created issues
+Labels
+App icon
+Deploying bots
+Resources
+Rules and Scoring System
+## Getting Started
```
Comment (ShivangiSingh17) [inline README.md:12]: In this "Getting Started" section can you also include about setting up the development environment. It really helps contributors to have a smooth start if they want to contribute to the project.
Also mention about the prerequisites if any.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,35 @@
...
+https://github.com/NJACKWinterOfCode/TranslateX/issues/6
+## Deploying Bots
+A special issue has been created for this purpose. Follow the link below:
+https://github.com/NJACKWinterOfCode/TranslateX/issues/7
+## Resources
+This app uses Firbase ML Kit's "On Device Translation"
 For more ,see https://developers.google.com/ml-kit/language/translation
+## Rules and Scoring System
+This project follows the rules and guidelines of NJACKWinterOfCode for allotting points. Check out the link below:
+https://github.com/NJACKWinterOfCode/Get-Started#rules-and-scoring-system
```
Comment (ShivangiSingh17) [inline README.md:35]: Also mention about the main libraries and the API's used.

**Review by ShivangiSingh17 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Mehulipal) [reply README.md:35]: I have done the changes,can you please review it?

File: README.md (hunk 1 above)
Comment (Mehulipal) [reply README.md:12]: I have made the changes,can you please review it?

**Pull request closed without merging**